src/lib/operators/jit_operator/operators/jit_expression.cpp

```diff
@@ -176,9 +176,17 @@ std::pair<const DataType, const bool> JitExpression::_compute_result_type() {
 
 template <typename ResultValueType>
 std::optional<ResultValueType> JitExpression::compute(JitRuntimeContext& context) const {
+  // Value ids are always retrieved from the runtime tuple
+  // NOLINTNEXTLINE(misc-suspicious-semicolon) clang tidy somehow does not like this constexpr
+  if constexpr (std::is_same_v<ResultValueType, ValueID>) {
```

Comment:
I do not understand why clang tidy does not like this constexpr:
````
/var/lib/jenkins/workspace/hyrise_hyrise_PR-1532/src/lib/operators/jit_operator/operators/jit_expression.cpp:180:59: error: potentially unintended semicolon [misc-suspicious-semicolon,-warnings-as-errors]
  if constexpr (std::is_same_v<ResultValueType, ValueID>) {
                                                          ^
````
[Jenkins Clang Tidy Error](https://hyrise-ci.epic-hpi.de/blue/organizations/jenkins/hyrise%2Fhyrise/detail/PR-1532/9/pipeline)

Reply:
Because it's broken. ;)

Reply:
https://reviews.llvm.org/D46027